nebula/ui/components/VPNConnectionInfoScreen.qml

```diff
@@ -8,7 +8,6 @@ import QtQuick.Layouts 1.14
 import Mozilla.VPN 1.0
 
 Rectangle {
-    property bool isLoading: false
```

Comment:
In theory, nebula should contain only reusable components. I cannot imagine this to be reused elsewhere.

Reply:
Right, I’ll move the QML components.